src/main/java/io/confluent/connect/elasticsearch/ElasticsearchClient.java

```diff
@@ -251,6 +253,7 @@ public void index(SinkRecord record, DocWriteRequest<?> request) {
         close();
       } catch (ConnectException e) {
         // if close fails, want to still throw the original exception
```

Comment:
```suggestion
        // if close fails, want to still log the original exception
```

Reply:
I think the comment is correct, we're logging the close exception, but in the line below, we're throwing the original exception.

Reply:
works for me indeed ! thank you